docs/fsharp/style-guide/formatting.md

```diff
@@ -116,378 +63,362 @@ This is sometimes called “vanity alignment” or “vanity indentation”. The
-            return ()
-        }
+// OK
+someFunction3
+    x.IngredientName1 x.Quantity2
+    x.IngredientName2 x.Quantity2
 
-let c =
-    { Name = "Bilbo"
-      Age = 111
-      Region = "The Shire" }
+// OK
+someFunction4
+    x.IngredientName1
+    x.Quantity2
+    x.IngredientName2
+    x.Quantity2
 
-let d =
-    while f do
-        printfn "%A" x
+// OK
+someFunction5
+    (convertVolumeToLiter x)
+    (convertVolumeUSPint x)
+    (convertVolumeImperialPint x)
 ```
 
-## Formatting blank lines
-
-* Separate top-level function and class definitions with two blank lines.
-* Method definitions inside a class are separated by a single blank line.
-* Extra blank lines may be used (sparingly) to separate groups of related functions. Blank lines may be omitted between a bunch of related one-liners (for example, a set of dummy implementations).
-* Use blank lines in functions, sparingly, to indicate logical sections.
-
-## Formatting comments
-
-Generally prefer multiple double-slash comments over ML-style block comments.
+When the function takes a single multi-line tupled argument, place each argument on a new line:
 
 ```fsharp
-// Prefer this style of comments when you want
-// to express written ideas on multiple lines.
+// OK
+someTupledFunction(
+    478815516,
+    "A very long string making all of this multi-line",
+    1515,
+    false
+)
 
-(*
-    ML-style comments are fine, but not a .NET-ism.
-    They are useful when needing to modify multi-line comments, though.
-*)
+// OK
+someTupledFunction
+    (478815516,
+     "A very long string making all of this multi-line",
+     1515,
+     false)
+)
 ```
 
-Inline comments should capitalize the first letter.
+If argument expressions are short, separate arguments with spaces and keep it in one line.
 
 ```fsharp
-let f x = x + 1 // Increment by one.
-```
-
-## Formatting string literals and interpolated strings
+let person = new Person(a1, a2)
 
-String literals and interpolated strings can just be left on a single line, regardless of how long the line is.
+let myRegexMatch = Regex.Match(input, regex)
 
-```fsharp
-let serviceStorageConnection =
-    $"DefaultEndpointsProtocol=https;AccountName=%s{serviceStorageAccount.Name};AccountKey=%s{serviceStorageAccountKey.Value}"
+let untypedRes = checker.ParseFile(file, source, opts)
 ```
 
-Multi-line interpolated expressions are strongly discouraged. Instead, bind the expression result to a value and use that in the interpolated string.
-
-## Naming conventions
+If argument expressions are long, use newlines and indent one level, rather than indenting to the left-parenthesis.
 
-### Use camelCase for class-bound, expression-bound, and pattern-bound values and functions
+```fsharp
+// OK
+let person =
+    new Person(
+        argument1,
+        argument2
+    )
 
-It is common and accepted F# style to use camelCase for all names bound as local variables or in pattern matches and function definitions.
+// OK
+let myRegexMatch =
```

Comment:
Should this not be something like
```fsharp
let myRegexMatch =
    Regex.Match(
        "my longer input string with some interesting content in it",
        "myRegexPattern"
    )
```
Seems a bit inconsistent otherwise.

Reply:
I'll change it for now.  However I don't think the guide is sufficiently clear on parenthesis location - for application of DUs it says either is ok, but this section implies
```fsharp
let myRegexMatch =
    Regex.Match(
        "my longer input string with some interesting content in it",
        "myRegexPattern"
    )
```
is normal.  What are the default fantomas settings?

Reply:
I've checked the fantomas default settings and documented them in the style guide.